webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderUserDetails.scala

```diff
@@ -100,10 +105,23 @@ object PasswordEncoder {
     override def encode(rawPassword: CharSequence): String = rawPassword.toString
     override def matches(rawPassword: CharSequence, encodedPassword: String): Boolean = rawPassword.toString == encodedPassword
   }
+  // Unsalted hash functions :
   val MD5    = new DigestEncoder("MD5"    )
   val SHA1   = new DigestEncoder("SHA-1"  )
   val SHA256 = new DigestEncoder("SHA-256")
   val SHA512 = new DigestEncoder("SHA-512")
+  // Salted hash functions :
+  val BCRYPT = new PasswordEncoder() {
+    override def encode(rawPassword: CharSequence): String = {
+      val random = new SecureRandom()
+      val salt: Array[Byte] = new Array(16)
+      random.nextBytes(salt)
+      OpenBSDBCrypt.generate("2b", rawPassword.toString.toCharArray, salt, RudderConfig.RUDDER_BCRYPT_COST)
```

Comment:
Please document the "2b" here, with a reference to `bcrypt` documentation and why we use that one.